application/templates/partials/_example.njk

```diff
@@ -50,7 +50,9 @@
       </ul>
 
       <div class="app-tabs__container js-tabs__container" id="{{ exampleId }}-html" role="tabpanel">
-        <a class="govuk-link app-link--copy" href="#copy" aria-live="assertive" data-module="copy-to-clipboard">Copy</a>
+        <div class="app-link--copy__container">
```

Comment:
If this is duplicated should it become a macro?

Reply:
TODO added - will pick this up as separate ticket